src/System.Linq.Queryable/src/System/Linq/Queryable.cs

```diff
@@ -842,6 +842,23 @@ public static TSource SingleOrDefault<TSource>(this IQueryable<TSource> source,
                     ));
         }
 
+        public static bool TrySingle<TSource>(this IQueryable<TSource> source, out TSource element)
+        {
+            var result = TrySingle(source);
```

Comment:
Nit: Use var only when it is clear from the line what is the type -- see https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md [10]